src/main/kotlin/blackjack/domain/participant/state/Stay.kt

```diff
@@ -1,3 +1,13 @@
 package blackjack.domain.participant.state
 
-class Stay(hand: Hand) : Finished(hand)
+import blackjack.domain.participant.GameResult
+
+class Stay(cards: Cards) : Finished(cards) {
+    override fun judgementGameResult(otherScore: Score): GameResult {
+        return when {
+            otherScore.isBlackjack -> GameResult.LOSE
+            otherScore.isBust -> GameResult.WIN
+            else -> this.score().compareGameResult(other = otherScore)
+        }
```

Comment:
GameResult를 구하는 부분의 역할이 Blackjack, Bust, Stay 등에서 각각 override 되고 있고 일부는 Score에서 구현되고 있는 것 같아요. Score에서 모든 역할을 수행할수도 있지 않을까요? 

compareGameResult에서 Bust 등의 모든 로직을 처리하고 아래와 같이 구현하는건 어떨까요?

```suggestion
        return this.score().compareGameResult(other = otherScore)
```

Reply:
이 부분에서 규남님과 같은 고민을 했었습니다!! 👍 👍 

Blackjack, Bust, Stay 등 상태에게 판단 역할을 넘긴 이유가 결국은 Blackjack 승리조건 때문이었는데요,
Score는 21점이 Blackjack에 해당되는 점수라는건 알지만, 21점이 무조건 Blackjack은 아니라는 상황 때문에 그랬습니다.

- Stay 21과 Blackjack 21 끼리의 승패를 비교하려면 카드의 장수를 알아야 함
- Stay와 Bust, Stay와 Blackjack 끼리의 승패를 비교하려면 점수 뿐만 아니라 상태값 까지 알아야함

결국 Score 쪽에서 상태와 카드 장수를 알아야하는 문제가 있어서 상태 쪽에 판단 로직을 넣고, Score는 점수만 비교하도록 했습니다.